jupyter-book/preprocessing_visualization/preprocessing.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>"It proved to be benefical and computationally efficient with software packages such as <em>scDblFinder</em> or <em>scds</em>. "</p><p>This reads a little bit weird.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>